docs/Contributing/Audit-logs.md

```diff
@@ -1302,6 +1302,44 @@ This activity contains the following fields:
 }
 ```
 
+## applied_spec_software
+
+Generated when a user edits the software of a team (or no team) via the fleetctl CLI.
+This activity contains the following fields:
+
+This activity contains the following fields:
+- "team_id": The ID of the team that the software applies to, null if they apply to devices that are not in a team.
+- "team_name": The name of the team that the software applies to, null if they apply to devices that are not in a team.
+
+#### Example
+
+```json
+{
+  "team_id": 123,
+  "team_name": "Workstations",
+  "software": {
```

Comment:
@eugkuo @iansltx I just pushed some tweaks. 

I think it would be nice if this matched the structure of `software` key in the YAML: https://fleetdm.com/docs/configuration/yaml-files#software 

What do y'all think?

Reply:
> Adding VPP into the mix would significantly affect level of effort

@iansltx by how many points you think?

Reply:
Minimum +3, since in order to avoid broadcasting an activity when no software was provided _but also_ broadcast an activity when software was deleted, we would need to merge VPP batch and software installer batch endpoints so we have a clear picture of before/after.

Additionally, right now fleetctl applies software packages and then VPP apps and without further digging I can't say whether that order of operations is significant. So either that order doesn't matter or it's an unknown-sized can of worms. Thinking more likely the latter since we're talking about GitOps code here.

Reply:
Re: the software key, if I'm consuming this JSON I'd rather iterate over one set of items and filter/null-check slug if I care about FMA, vs. iterating over two nearly identical keys (I had thought about splitting FMA out).

Reply:
@iansltx I think let's keep VPP in this pass. So we can get it all the way done while we're at it.

> Re: the software key, if I'm consuming this JSON

I'll defer to you and @rachaelshaw (API design DRI) here. Please feel free to update the PR with your proposed changes.